tools/go-cli/build.gradle

```diff
@@ -54,15 +19,16 @@ task distBinary(dependsOn: [removeBinary, distDocker]) << {
     run(["mkdir", "${projectDir}/../../bin/go-cli/"])
     run(dockerBinary + ["run", "--name", dockerContainerName, dockerImageName + ":" + dockerImageTag])
 
-    // Copy all Go binaries outside of Docker into openwhisk go-cli/bin/ folder
+    // Copy all Go binaries outside of Docker into openwhisk go-cli/build/ folder
     run(dockerBinary + ["cp", dockerContainerName +
             ":/src/github.com/go-whisk-cli/build/.", "${projectDir}/../../bin/go-cli"])
 
-    if (ant.properties.family in ["linux", "mac"] && ant.properties.arch in ["amd64", "386"]) {
-        exec {
-            executable "ln"
-            args "${projectDir}/../../bin/go-cli/${ant.properties.family}/${ant.properties.arch}/wsk", "${projectDir}/../../bin/go-cli/wsk", "-s"
-        }
+    // Create a shorthand sym link to current platform appropriate Go CLI binary
+    String go_osname = mapOsNameToGoName(getOsName())
+    String go_osarch = mapOsArchToGoArch(getOsArch())
+    exec {
+        executable "ln"
+        args "${projectDir}/../../bin/go-cli/"+go_osname+"/"+go_osarch+"/wsk", "${projectDir}/../../bin/go-cli/wsk", "-s"
```

Comment:
@markusthoemmes - Thanks for taking a look.  Are you thinking of the groovy simple templating or something similar?  I like the readability as it is right now; but if there is some other benefit from templates I'm missing, would you please elaborate.